native/src/program/state.rs

```diff
@@ -57,6 +57,11 @@ where
         &self.program
     }
 
+    /// Returns a reference to the [`Program`] of the [`State`].
+    pub fn program_mut(&mut self) -> &mut P {
```

Comment:
sorry this was accident; will remove if this PR is good idea. i will think i will need a `program_mut` though, for unrelated reason :)